tests/acceptance/features/apiOcm/share.feature

```diff
@@ -1212,7 +1145,47 @@ Feature: an user shares resources using ScienceMesh application
       | imageFolder  |
     Then the HTTP status code should be "200"
     And the downloaded zip archive should contain these files:
-      | name                       | content    |
-      | textfile.txt               | some data  |
-      | imageFolder                |            |
-      | imageFolder/testavatar.png |            |
+      | name                       | content   |
+      | textfile.txt               | some data |
+      | imageFolder                |           |
+      | imageFolder/testavatar.png |           |
+
+  @issue-11033
+  Scenario: sharee shouldn't be able to update file when share expires
+    Given using SharingNG
+    And using server "LOCAL"
+    And user "Alice" has uploaded file with content "ocm test" to "/textfile.txt"
+    And user "Alice" has sent the following resource share invitation to federated user:
+      | resource        | textfile.txt |
+      | space           | Personal     |
+      | sharee          | Brian        |
+      | shareType       | user         |
+      | permissionsRole | File Editor  |
+    When user "Alice" expires the last share of resource "textfile.txt" inside of the space "Personal"
+    Then the HTTP status code should be "200"
+    And using server "REMOTE"
+    When user "Brian" updates the content of federated share "textfile.txt" with "this is a new content" using the WebDAV API
+    Then the HTTP status code should be "403"
```

Comment:
IMO, 404 would be perfect here. but that can change as per the server implementation
```suggestion
    Then the HTTP status code should be "404"
```